projects/fusion-ui/components/radio-group/v4/radio-group.base.component.ts

```diff
@@ -0,0 +1,127 @@
+import {EventEmitter, Input, OnInit, Output, ChangeDetectorRef, Directive} from '@angular/core';
+import {ControlValueAccessor} from '@angular/forms';
+import {LogService} from '@ironsource/fusion-ui/services/log';
+import {UniqueIdService} from '@ironsource/fusion-ui/services/unique-id';
+import {RadioGroupOptions} from '@ironsource/fusion-ui/components/radio-group/common/entities';
+
+@Directive()
+export abstract class RadioGroupBaseComponent implements OnInit, ControlValueAccessor {
+    /** @internal */
+    id: string; // radio buttons group id
+
+    /**
+     * radio buttons group name
+     */
+    @Input() set name(value: string) {
+        this._name = value;
+    }
+    /**
+     * radiobutton inline direction
+     */
+    @Input() inline = false;
+    /**
+     * disabled (all group)
+     */
+    @Input() set disabled(value: boolean) {
+        this._disabled = value;
+    }
+    /**
+     * radiobutton option: interface RadioGroupOptions {
```

Comment:
If we don't need it lets remove this from the code

Reply:
I will remove interface DTO